plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/contributors/FirCallableCompletionContributor.kt

```diff
@@ -94,24 +101,32 @@ internal open class FirCallableCompletionContributor(
         val (implicitScopes, implicitReceivers) = implicitScopesContext
         val implicitReceiversTypes = implicitReceivers.map { it.type }
 
-        val availableNonExtensions = collectNonExtensions(implicitScopes, visibilityChecker, scopeNameFilter) { filter(it) }
+        val availableNonExtensions =
+            collectNonExtensions(implicitScopes, visibilityChecker, scopeNameFilter) { filter(it) }
         val extensionsWhichCanBeCalled = collectSuitableExtensions(implicitScopes, extensionChecker, visibilityChecker)
 
-        availableNonExtensions.forEach { addCallableSymbolToCompletion(expectedType, it, getOptions(it)) }
+        availableNonExtensions.forEach {
+            addCallableSymbolToCompletion(
+                expectedType,
+                it,
+                getOptions(it) ?: return@forEach,
+                getPriority(it)
+            )
+        }
         extensionsWhichCanBeCalled.forEach { (symbol, applicable) ->
-            getExtensionOptions(symbol, applicable)?.let { addCallableSymbolToCompletion(expectedType, symbol, it) }
+            getExtensionOptions(symbol, applicable)?.let { addCallableSymbolToCompletion(expectedType, symbol, it, getPriority(symbol)) }
         }
 
         if (shouldCompleteTopLevelCallablesFromIndex) {
             val topLevelCallables = indexHelper.getTopLevelCallables(scopeNameFilter)
             topLevelCallables.asSequence()
                 .map { it.getSymbol() as KtCallableSymbol }
                 .filter { with(visibilityChecker) { isVisible(it) } }
-                .forEach { addCallableSymbolToCompletion(expectedType, it, getOptions(it)) }
+                .forEach { addCallableSymbolToCompletion(expectedType, it, getOptions(it) ?: return@forEach, getPriority(it)) }
         }
 
         collectTopLevelExtensionsFromIndices(implicitReceiversTypes, extensionChecker, visibilityChecker)
-            .forEach { addCallableSymbolToCompletion(expectedType, it, getOptions(it)) }
+            .forEach { addCallableSymbolToCompletion(expectedType, it, getOptions(it) ?: return@forEach, getPriority(it)) }
```

Comment:
Please, do not use inline return/break/continue statements. Introduce a variable for that